projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -128,14 +130,14 @@ export class DisplayEditComponent implements OnInit {
 
         this.readOnlyValue = this._valueService.isReadOnly(this.valueTypeOrClass, this.displayValue);
 
-        this._dspApiConnection.admin.usersEndpoint.getUserByIri(this.displayValue.attachedToUser).subscribe(
-            (response: ApiResponseData<UserResponse>) => {
-                this.user = response.body.user;
-            },
-            (error: ApiResponseError) => {
-                console.error(error);
-            }
-        );
+        // prevent getting info about system user (standoff link values are managed by the system)
+        if (this.displayValue.attachedToUser !== 'http://www.knora.org/ontology/knora-admin#SystemUser') {
```

Comment:
@mdelez This prevents requesting user info for the system user. Is this fine (the `user` prop) remains `undefined`.

Reply:
The info box does not work properly for the system user:

![Screenshot 2020-11-02 at 14 28 12](https://user-images.githubusercontent.com/6000023/97873270-aa16b400-1d17-11eb-8154-84a2e9b7fd38.png)

Reply:
fixed in https://github.com/dasch-swiss/dsp-ui-lib/pull/217/commits/089c5af762049cd44c475676127839fbf5aab9fa